net/frr/src/opnsense/service/templates/OPNsense/Quagga/frr

```diff
@@ -16,3 +16,9 @@ start_postcmd="/usr/local/opnsense/scripts/frr/carp_event_handler"
 {% else %}
 frr_enable="NO"
 {% endif %}
+{% if OPNsense.quagga.general.enablesnmp == '1' %}
+zebra_flags="${zebra_flags} -M snmp"
```

Comment:
We add this to zebra_flags but zebra.conf was not modified?

Reply:
Zebra.conf was an oversight - that is now fixed.